src/cfe.c

```diff
@@ -491,6 +503,37 @@ else
   *surface_runoff_depth_m = 0.0;
   *infiltration_depth_m = 0.0;
   }
+
+
+ // impermeable fraction due to frozen soil
+ double factor = 1.0;
+ //field_capacity_m = SMCREF in NOAH_MP
+ // ice_content_threshold = frzk in NOAH_MP
+ //double frzk = 0.15; // Ice content above which soil is impermeable 
+ int cv_frz = 3; // should this be moved to config file as well, probably not.

```

Comment:
I don't see that cv_frz is used outside the if statement.  Can we initialize within the if statement below?

Reply:
done